pkg/apply/task/inv_add_task.go

```diff
@@ -71,25 +134,51 @@ func (i *InvAddTask) Cancel(_ *taskrunner.TaskContext) {}
 // StatusUpdate is not supported by the InvAddTask.
 func (i *InvAddTask) StatusUpdate(_ *taskrunner.TaskContext, _ object.ObjMetadata) {}
 
-// inventoryNamespaceInSet returns the the namespace the passed inventory
+// NamespaceInSet returns the the namespace the passed inventory
 // object will be applied to, or nil if this namespace object does not exist
 // in the passed slice "infos" or the inventory object is cluster-scoped.
-func inventoryNamespaceInSet(inv inventory.Info, objs object.UnstructuredSet) *unstructured.Unstructured {
-	if inv == nil {
-		return nil
-	}
-	invNamespace := inv.Namespace()
-
+func NamespaceInSet(inv client.Object, objs object.UnstructuredSet) *unstructured.Unstructured {
```

Comment:
inventoryNamespaceInSet -> NamespaceInSet rename could be pulled out to a rename/refactor PR.